### PR TITLE
Create a data model

### PR DESCRIPTION
**Pull request by jrosario19**
In this PR I added the following features:

Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220809133409_add_user_ref_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220809133409_add_user_ref_to_posts.rb]: - [ ] Good job adding reference to the foreign key but according to the ERD diagram below, the user table should be referenced with an alias of `author`. Please kindly implement this.

File: db/migrate/20220809133750_add_user_ref_to_comment.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220809133750_add_user_ref_to_comment.rb]: - [ ] Kindly see above feedback and also implement same here.

File: db/migrate/20220809134122_create_likes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (AnselemOdims) [inline db/migrate/20220809134122_create_likes.rb]: - [ ] Kindly see above feedback and also implement same here.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @jrosario19, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/migrate/20220809133409_add_user_ref_to_posts.rb (hunk 1 above)
Comment (jrosario19) [reply db/migrate/20220809133409_add_user_ref_to_posts.rb]: Hi @AnselemOdims Thank you for your kind response. I have one question, should I recreate the migrations or just modify the existing ones?

**Review by Davidosky007 — APPROVED** [review]
Hi @jrosario19 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well wriiten readme file 👍 
* Good looking PR description 👍 
* Proper use of Gitflow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**